### DISCUSSION TITLE
packaging scene

### DISCUSSION THREAD

**Discussion by yapsgg** [ExtendRealityLtd/VRTK.Tutorials.VRBowling#25] [General]
Hello, bro i have a question. How can i convert my Unity project to scene so i when i open with another laptop i should be able to just run it no create again using the models.

**Comment by jhoncrazu412** [comment]
Hey there! If you want to convert your Unity project into a standalone scene, you can use the "Build Settings" option. Just go to File -> Build Settings, choose your target platform, and click on "Build." This will create an executable file that you can run on any other laptop without the need to recreate the models or assets. Happy [Packaging](https://packaginglane.com/)!
Keywords: Unity project, convert, scene, laptop, run, models, packaging.
